.swiftformat

```diff
@@ -0,0 +1,16 @@
+# file options
+
+--swiftversion 5.1
```

Comment:
Do we want to use 5.2 here as that’s the version we use in `Package.swift` as well?

Reply:
lemme check, can't remember why we still have 5.1 here even though our projects are 5.2 hm :)